src/storage/src/protocol/client.rs

```diff
@@ -156,14 +158,44 @@ impl RustType<ProtoIngestSourceCommand> for IngestSourceCommand<mz_repr::Timesta
             description: proto
                 .description
                 .into_rust_if_some("ProtoIngestSourceCommand::description")?,
-            resume_upper: proto
-                .resume_upper
-                .map(Into::into)
-                .ok_or_else(|| TryFromProtoError::missing_field("ProtoCompaction::resume_upper"))?,
+            resume_upper: proto.resume_upper.map(Into::into).ok_or_else(|| {
+                TryFromProtoError::missing_field("ProtoIngestSourceCommand::resume_upper")
+            })?,
+        })
+    }
+}
+
+impl RustType<ProtoExportSinkCommand> for ExportSinkCommand<mz_repr::Timestamp> {
+    fn into_proto(&self) -> ProtoExportSinkCommand {
+        ProtoExportSinkCommand {
+            id: Some(self.id.into_proto()),
+            description: Some(self.description.into_proto()),
+            resume_upper: Some((&self.resume_upper).into()),
+        }
+    }
+
+    fn from_proto(proto: ProtoExportSinkCommand) -> Result<Self, TryFromProtoError> {
+        Ok(ExportSinkCommand {
+            id: proto.id.into_rust_if_some("ProtoExportSinkCommand::id")?,
+            description: proto
+                .description
+                .into_rust_if_some("ProtoExportSinkCommand::description")?,
+            resume_upper: proto.resume_upper.map(Into::into).ok_or_else(|| {
+                TryFromProtoError::missing_field("ProtoExportSinkCommand::resume_upper")
+            })?,
         })
     }
 }
 
+/// A command that starts ingesting the given ingestion description
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct ExportSinkCommand<T> {
+    pub id: GlobalId,
+    pub description: SinkDesc<CollectionMetadata, T>,
+    /// The upper frontier that this ingestion should resume at
```

Comment:
```suggestion
    /// The upper frontier at which this export should resume.
```

Reply:
whoops thanks -- will update comments on this struct to avoid dangling prepositions